1-react/store/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { createStore } from './slomux'
+
+const CHANGE_INTERVAL = 'CHANGE_INTERVAL'
+
+const reducer = (state, action) => {
+  switch(action.type) {
+    case CHANGE_INTERVAL:
+      return state += action.payload
```

Comment:
Позволять устанавливать отрицательные значения не имеет смысла 😉.

```suggestion
      return Math.max(0, state + action.payload)
```